src/autocapture-utils.ts

```diff
@@ -338,14 +338,16 @@ export function getDirectAndNestedSpanText(target: Element): string {
  */
 export function getNestedSpanText(target: Element): string {
     let text = ''
-    for (const child of target?.children || []) {
-        if (child && child.nodeType === 1 && child.tagName.toLowerCase() === 'span') {
-            const spanText = getSafeText(child)
-            if (shouldCaptureValue(spanText)) {
-                text = concatenateStringsWithSpace([text, spanText])
-            }
-            if (child.children.length > 0) {
-                text = concatenateStringsWithSpace([text, getNestedSpanText(child)])
+    if (target.children?.length > 0) {
+        for (const child of target?.children) {
```

Comment:
```suggestion
    if ((target?.children || []).length > 0) {
        for (const child of target.children) {
```

nit picking, since we've got `target.children` on the line above we don't need `target?.children` here.

If `target` is undefined it'll fail on line 341. 

(sorry, if my last nit-pick here dig a hole for you with IE 😅)

Reply:
It's weird because that change you suggested initially passed the IE test, but then failed subsequently, but reverting it made it pass again 🤷‍♀️ I'll update kinda as you suggest here (I'd argue that the way I did it is overall more readable) but if it fails after I make the commit it'll be your fault 😛

Reply:
🤣 

> (I'd argue that the way I did it is overall more readable)

So, there meaning

`if (target.children?.length > 0)`

vs.

`if ((target?.children || []).length > 0)`

Yep, we're fighting JS (and I did say I was nit-picking so feel free to ignore)

Even though TS says `target` can't be undefined JS won't care so it can be. Which would mean `target.children` could fail. Before you get to the `target.children?.length`. A little over cautious but the SDK is a pain for hitting all the possible errors.

I guess you could have `if(target && target.children?.length > 0)` instead.

If it's not undefined then the spec says we definitely get children... But, JS is fun so I won't quibble over the `children?.length` not needing the elvis operator since it doesn't cost us anything.

(Going into detail in case I'm being stupid and this helps you figure out whether to ignore me :))

Reply:
No, you're not being stupid, I guess I'm just being overly trusting of typescript haha. I hit the merge button too quick. I'll open another PR and it can go out with the next version.